pages/index.js

```diff
@@ -11,4 +12,4 @@ function Home() {
   );
 }
 
-export default withReduxSaga(Home, { type: 'GET_REPO_SERVER' });
+export default withReduxSaga(configureStore, Home, { type: 'GET_REPO' });
```

Comment:
I think I should expand this API to mimic `withRedux`.  that means

`withReduxSaga(configObject, mapStateToProps, mapDispatchToProps, mergeProps, connectOptions`. Those last 3 are optional just like using `withRedux` (or `connect`).

I need to think about this.